domain/lib/src/usecases/biometric_auhentication/authentication_biometric_interactor.dart

```diff
@@ -45,6 +45,8 @@ class AuthenticationBiometricInteractor {
     } catch (exception) {
       if (exception is BiometricNotEnrolled) {
         return Right<Failure, Success>(AuthenticationBiometricViewState(AuthenticationBiometricState.unEnrolled));
+      } else if (exception is BiometricLockedOut) {
+        return Right<Failure, Success>(AuthenticationBiometricViewState(AuthenticationBiometricState.locked));
```

Comment:
Why you prefer to catch an exception with a Right value. Is it make sense?

Reply:
Well it helps me to check some of the cases that happen when biometric authentication does not have a function that specifically checks those cases.